uint8 start_vision
---
bool received
